src/utils/axios.ts

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: process.env.API_BASE_URL,
```

Comment:
I'm using `EnvironmentPlugin` to expose these values through `process.env`. This is useful because Jest doesn't have access to these values with the way Vite does it by default.

Reply:
Sounds good.